### DISCUSSION TITLE
Link does not exist - when creating profile

### DISCUSSION THREAD

**Discussion by magge-faf** [audiamus/BookLibConnect#118] [Q&A] [answered]
When I open the URL (after hitting the 'Create login URL') and enter my credentials and successfully log in, I just get "link cannot be found" (see image)

![image](https://github.com/audiamus/BookLibConnect/assets/101107758/e626270e-a90f-4d57-adf1-d0e1f09dab91)

So, any idea what I am doing wrong? I am from Germany and selected Region Germany

[BookLibConnect_2023-09-25_001.log](https://github.com/audiamus/BookLibConnect/files/12710852/BookLibConnect_2023-09-25_001.log)

**Comment by audiamus** [comment, marked as answer]
The question should be marked FAQ number 1. Everything is fine. Just follow the instructions and copy this final URL to the empty text field in the Book Lib Connect dialog.

**Reply by brunoalbertopeixoto** [reply]
@audiamus 
I'm Brazilian and I'm having the same problem. I don't understand what procedure I should do to make the account work. Which URL should I copy when this error appears:
https://www.[amazon.com.br/ap/404](https://www.amazon.com.br/ap/404)
![image](https://github.com/audiamus/BookLibConnect/assets/41241891/07f3dfae-8bba-45b3-b135-5f9d8f2f6c0f)
![image](https://github.com/audiamus/BookLibConnect/assets/41241891/6f8f34ed-3372-4985-bc74-1cd5bc6cba25)

**Reply by audiamus** [reply]
I believe it's a bit more complicated with the Brazil marketplace. I'm not able to test this myself and can only refer you to the discussion we had here: https://github.com/audiamus/BookLibConnect/issues/117

**Reply by brunoalbertopeixoto** [reply]
@audiamus I don't understand what I have to do. I'm a layman on this subject. Could you explain it in a more didactic way? How do I get my 8 hex code?

**Reply by audiamus** [reply]
As I said, I can't test this myself for the Brazil marketplace, hence I cannot explain the procedure. The way I understand it you have to intercept the HTTP traffic between your browser and the Audible server, to avoid redirection and to grab the URL that contains the access token.

With the method used by BookLibConnect and other modern 3rd party clients (and the Audible apps themselves), there is no 8 digit hex code any more. Instead, the access token provided after the initial successful login will grant you access to your library. All the magic with vouchers, keys and converting to plain audio will be handled by BookLibConnect and AAXClean, the underlying decryption library.

**Comment by magge-faf** [comment]
My bad, thank you - I cannot believe I have not seen that text. Maybe color it red because the grey text looks kind of unimportant and gets automatically ignored by the user's eye IMO. However, it works now. Excellent job on the software, tho.

**Comment by dustinhartlyn** [comment]
I know it is simple, but is not obvious. I think the instruction should be in Bold for the less observant among us. After all, those instructions are what the user is wanting. For daft people like me who spent way to long trying to figure out what to do... here is what you do link you audible account to this amazing little program:

1) Click "Profiles" button on main screen and select to add profile
2) Select your country. Note that I am in the USA and have only tested it there.
3) Click "Create Login URL"
4) Open that URL in a web browser and login to your audible account. The website will say that the link is not valid... however the URL has been changed. 
5) After logging in, copy the URL in the web browser where it says "Looking for something? ......." and paste it into the lower text box of Book Lib Connect bellow where the first link was generated. It will automatically test the link and verify that it works.

**Comment by topse** [comment]
Same problem in region germany:

![grafik](https://github.com/audiamus/BookLibConnect/assets/893695/3b0647d2-b9c8-4b20-af3d-a8293f43f945)

I would appreciate any hint what to do.

Thanks for your great work and Best Regards,
Tobias

**Comment by topse** [comment]
I am so sorry :-/ It works perfectly!

Best Regards and thanks for the great work!!!
Tobias
